bot/src/slack.js

```diff
@@ -48,11 +78,31 @@ const updateMessage = async (event) => {
     }
   })
 
-  await upsertReport({ts, message})
+  const tags = getTags(message)
+
+  if (tags.length) {
+    if (thread_ts) {
+      if (thread_ts === ts) {
```

Comment:
Can you please remind me when is `thread_ts` not set? (or what is the difference between `thread_ts` not set and `thread_ts === ts`)?

Reply:
It's not set on individual messages (not part of threads). `thread_ts === ts` means it's the root message of a thread .

Reply:
...and when someone replies to that message it becomes a thread, right?

Reply:
Yes.